### PR TITLE
Unfeature flag the cloud debugger

### PR DESCRIPTION
**Pull request by patflynn**
Relates to #1032.

### CODE REVIEW

**Comment by etanshaul** [comment]
Are you planning on submitting individual PRs to unfeature each component?

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/startup/CloudToolsPluginInitializationComponent.java [hunk 1]
```diff
@@ -62,10 +62,6 @@ public void initComponent() {
     CloudToolsPluginInfoService pluginInfoService = ServiceManager
         .getService(CloudToolsPluginInfoService.class);
 
-    if (pluginInfoService.shouldEnable(GctFeature.DEBUGGER)) {
```
Comment (etanshaul) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/startup/CloudToolsPluginInitializationComponent.java]: Should we also remove `GctFeature.Debugger`?

**Comment by patflynn** [comment]
yes to both comments.

**Comment by etanshaul** [comment]
ok. lgtm then after the unit test fix

**Pull request merged**